content/ai/concepts/search algorithms/best first search.md

```diff
@@ -0,0 +1,28 @@
+---
+Title: 'Best-First Search'
+Description: 'A graph searching algorithm that expands a node based on an evaluation function.'
```

Comment:
```suggestion
Description: 'A graph searching algorithm that determines a path between nodes based on an evaluation function.'
```

Reply:
I just found "expands a node" very abstract and opaque. I've provided an alternative, but feel free to try something else.